packages/netlify-cms-locales/src/index.js

```diff
@@ -2,6 +2,7 @@ export { default as cs } from './cs';
 export { default as de } from './de';
 export { default as en } from './en';
 export { default as es } from './es';
+export { default as cat } from './cat';
```

Comment:
Shouldn't we use the two letter code like the rest? e.g. `ca`

Reply:
Yes you're right if we use 2 [letter ISO code](https://www.sitepoint.com/iso-2-letter-language-codes/) it should be `ca `(instead of `cat`).  I've updated the PR with this new change. 
Thanks for the advice.